src/main/java/org/openmrs/addonindex/domain/AddOnInfoAndVersions.java

```diff
@@ -192,4 +192,10 @@ public void addTag(String tag) {
 		}
 		tags.add(tag);
 	}
+
+	public void setDetailsBasedOnLatestVersion(){
+        AddOnVersion version = getVersion(getLatestVersion()).orElseThrow(IllegalArgumentException::new);
```

Comment:
If we're going to throw an exception it should be IllegalStateException. But actually, let's just return in this case. There's no need to fail here.